src/wishlist.tm

```diff
@@ -21,7 +21,71 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
+
+  <hrule>
+
+  Being able to easily create macros, at least non-parametrical ones, with a
+  single menu action. It is already possible, but the syntax not easy for
+  begginers (and non-programmers in general).
+
+  <hrule>
+
+  Being able to adjust the math typesetting parameters. For example, by
+  default, I find formulas too compact, and I really like to have more space
+  around (in)equality signs, so I always include 0.2em spaces around them.
+  Looks nice, but semantically horrible.
+
+  <hrule>
+
+  A variant for underbraces and overbraces with <verbatim|\\smash> ;
+  currently <verbatim|\<less\>resize\<gtr\>> can be used but it is slow to
+  use and content can't be really modified afterwards.
+
+  <hrule>
+
+  Configuration option for pasting / drag-dropping images as integrated
+  images, not mere links to external file.
+
+  <hrule>
+
+  More complete menus : some objects are accessible only by keyboard, and
+  some took litteraly years to find them by accident or by chance in the
+  documentation.
+
+  <hrule>
+
+  A set of shortcuts for common colors (red, dark green, blue, grey).
+
+  <hrule>
+
+  A set of shotcuts for commonly used unicode symbols, such as
+  \<Rightarrow\>, \<rightarrow\>, <text-dots>
+
+  <hrule>
+
+  Performance : Editing large documents is a pain on a not-very-recent
```

Comment:
I would say that these are mostly bugs which should be filed in the buglist.

Reply:
Joris already said it is a goal, but this need deep refactoring on TeXmacs' core, which will probably not tomorrow, especially because it's fine with "recent" computers. Hence a wish rather than a bug.